RubeesEat/Model/InMemoryBillRepository.cs

```diff
@@ -21,6 +21,10 @@ public IReadOnlyList<Bill> GetAll()
 
     public Bill? GetById(Guid guid)
     {
+        if (!_bills.ContainsKey(guid))
+        {
+            return null;
+        }
```

Comment:
revert please, throws an exception anyways, which is more intuitive. Also change the return type to Bill

Reply:
In the other branch it also had Bill? as return type, but couldnt even return null (would only throw an exception)

Reply:
but yeah will do 👍